applications/OptimizationApplication/python_scripts/utilities/helper_utilities.py

```diff
@@ -0,0 +1,27 @@
+from importlib import import_module
```

Comment:
Please consult my general comments. The most important point is to remove optimization_info from the analysis stuff. optimization_info is about optimization things not analysis details.

Reply:
addressed in general answer